### PR TITLE
include patient id with questionnaire response

### PR DESCRIPTION
**Pull request by KeeyanGhoreshi**
fixes saving the questionnaire response on logica health.  The 400 error was due to a the "unknown" patient and practitioner ID.  Goes along with the fix to POST requests after switching the proxy.

### CODE REVIEW

**Comment by yuliumitre** [comment]
@KeeyanGhoreshi Is deploying to Logica Health the only way to test this PR?

**Comment by KeeyanGhoreshi** [comment]
you can test this PR by standalone launching the app with the logica health fhir server as the `iss`.  You can run the DTR app locally.  You can standalone launch by going to http://localhost:3005/launch?iss=https://api.logicahealth.org/MitreTest/data

When the app loads, try to `Save` the questionnaire response.  On master it will throw a 400 error.

File: src/components/QuestionnaireForm/QuestionnaireForm.jsx [hunk 1]
```diff
@@ -698,10 +698,13 @@ export default class QuestionnaireForm extends Component {
     var qr = window.LForms.Util.getFormFHIRData('QuestionnaireResponse', 'R4', "#formContainer");
     console.log(qr);
     qr.status = status;
-    qr.author = {
-      reference:
-        this.getPractitioner()
-    };
+    if (this.getPractitioner() !== "Unknown") {
```
Comment (yuliumitre) [inline src/components/QuestionnaireForm/QuestionnaireForm.jsx:701]: What do we do if this.getPracticitioner() is "Unknown"?  qr.author.reference will not be set?

Comment (KeeyanGhoreshi) [reply src/components/QuestionnaireForm/QuestionnaireForm.jsx:701]: yes it's not set at all

**Review by yuliumitre — APPROVED** [review]
The changes work well and the error is gone. Thanks.

**Pull request merged**